### ISSUE TITLE
No orders are placed

### ISSUE THREAD

**Issue by coderlv** [jufkes/kucoinBot#1]
there are enough balance in ETH and in DBC coin to place orders, but i got this:
2018-01-04 12:01:21,163 INFO     setting buy of 1216.6731 at 0.00014005999999999998
2018-01-04 12:01:21,460 INFO     None
2018-01-04 12:01:21,461 INFO     setting sell of 1168.0062 at 0.00015794
2018-01-04 12:01:21,757 INFO     None

sometimes sell orders are placed, but for buy still got None:
2018-01-04 11:50:19,717 INFO     Waiting 30 for next cycle...
2018-01-04 11:50:32,673 INFO     No orders present...setting to ticker price
2018-01-04 11:50:33,315 INFO     setting buy of 1250.0064 at 0.00014399999999999998
2018-01-04 11:50:33,638 INFO     None
2018-01-04 11:50:33,638 INFO     setting sell of 1200.0062 at 0.000156
2018-01-04 11:50:33,985 INFO     {'orderOid': '5a4df8e919d602695d3b8845'}
2018-01-04 11:50:33,985 INFO     Waiting 30 for next cycle...

**Comment by coderlv** [comment]
issue was in ETH price precision it's 6

**Comment by kingsofcoms** [comment]
is it working now? does it work better with high volume pairs or vice versa.

**Comment by jufkes** [comment]
@coderlv  -- Do I need to make a change in the price setting and force precision to 6 places? I've noticed that kucoin's API returns data structures kind strangely. It would not surprise me if they have inconsistent data returns.

@kingsofcoms  I've only used the bot on UTK and VEN. i have buy / sell value percentages set to 6, buyDiff 2, volume 6 and it's doing well. 

This is a partial set of UTK Dealt:

UTK / BTC | 2018-01-03 23:11:35 | Buy | 0.00005342BTC | 478.9595UTK | 0.02558602BTC
UTK / BTC | 2018-01-03 21:52:16 | Sell | 0.00005683BTC | 451.8733UTK | 0.02567996BTC
VEN / BTC | 2018-01-03 21:51:23 | Sell | 0.0002333BTC | 28.1051VEN | 0.00655692BTC

It does not like low values for the buy / sell. I initially ran it with 4 and, about 10% of the time, buy high then sell low. With 6's, i havent seen that.
